tests/test_runner.py

```diff
@@ -0,0 +1,80 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+import os
+
+import pytest
+from pytest_mock import mocker  # noqa F401
+
+from galaxy_importer import config
+from galaxy_importer.ansible_test import runners
+
+
+@pytest.fixture
+def temp_config_file():
+    try:
+        dir = os.path.dirname(os.path.dirname(__file__))
+        config_file = os.path.join(dir, 'galaxy_importer', 'galaxy-importer.cfg')
+        yield config_file
+    finally:
+        os.remove(config_file)
+
+
+def test_get_runner_no_config_file():
+    config.Config.IS_LOADED = False
```

Comment:
config.Config.IS_LOADED is false by default isnt it?  It shouldn't need to be set before the tests.  If it is getting changed at runtime by the config.Config.__init__() that is a symptom of the complexity of changing class variables dynamically.

Reply:
the config.Config.IS_LOADED state is maintained across tests, so it is set here to false to force config to reload.